dmscripts/generate_supplier_user_csv.py

```diff
@@ -32,6 +32,7 @@ def generate_supplier_csv(framework_slug, data_api_client, logger):
         "application_result",
         "framework_agreement",
         "variations_agreed",
+        "other_company_registration_number",
```

Comment:
Could we put it next to `companies_house_number` so it's easy to see on the CSV which one the supplier has?

Reply:
Yeah, I did think about this but was concerned that if Dave or Ash have macros working off this spreadsheet they might get misaligned, what do you think?